### PR TITLE
no-git for publish jobs, empty artifacts dir

### PR DESCRIPTION
**Pull request by TriplEight**

### CODE REVIEW

**Comment by TriplEight** [comment]
first run nightly on it

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -119,9 +127,10 @@ publish-snap:
   allow_failure:                   true
   <<:                              *collect_artifacts
 
-publish-onnet-update:
+publish-onchain-update:
   stage:                           publish-onchain
```
Comment (5chdn) [inline .gitlab-ci.yml]:
```suggestion
  stage:                           publish
```

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -119,9 +127,10 @@ publish-snap:
   allow_failure:                   true
   <<:                              *collect_artifacts
 
-publish-onnet-update:
+publish-onchain-update:
```
Comment (5chdn) [inline .gitlab-ci.yml]:
```suggestion
publish-onchain:
```

Comment (TriplEight) [reply .gitlab-ci.yml]: :+1:

File: .gitlab-ci.yml (hunk 1 above)
Comment (TriplEight) [reply .gitlab-ci.yml]: :+1:

File: .gitlab-ci.yml [hunk 3]
```diff
@@ -94,6 +99,7 @@ build-windows:
 publish-docker:
   stage:                           publish
   only:                            *releaseable_branches
+  <<:                              *no_git
```
Comment (gabreal) [inline .gitlab-ci.yml]: I guess you need to get the `Dockerfile` somehow.
https://github.com/paritytech/substrate/blob/kubernetes/.gitlab-ci.yml#L143

Comment (TriplEight) [reply .gitlab-ci.yml]: decided to stay with fetch strategies for a while.

**Pull request merged**